src/common/Messages.h

```diff
@@ -39,15 +39,16 @@ enum DSInstructionType : unsigned char
     SETPRIMARY = 0x00,
     POW1SUBMISSION = 0x01,
     DSBLOCKCONSENSUS = 0x02,
-    POW2SUBMISSION = 0x03,
-    SHARDINGCONSENSUS = 0x04,
-    MICROBLOCKSUBMISSION = 0x05,
-    FINALBLOCKCONSENSUS = 0x06,
-    AllPoWConnRequest = 0x07,
-    AllPoWConnResponse = 0x08,
-    LastDSBlockRequest = 0x09,
-    LastDSBlockResponse = 0x0A,
-    VIEWCHANGECONSENSUS = 0X0B
+    SHARDINGCONSENSUS = 0x03,
+    MICROBLOCKSUBMISSION = 0x04,
+    FINALBLOCKCONSENSUS = 0x05,
+    AllPoWConnRequest = 0x06,
+    AllPoWConnResponse = 0x07,
+    LastDSBlockRequest = 0x08,
+    LastDSBlockResponse = 0x09,
+    VIEWCHANGECONSENSUS = 0X0A,
+    AllPoW2Request = 0x0B,
+    AllPoW2Response = 0x0C,
```

Comment:
There's a naming consistency issue here, although the problem was not introduced in this PR. We can still fix them all this time